src/prospectScraper/Program.cs

```diff
@@ -121,12 +151,37 @@ private static void RunTheMockDraft()
             {
                 UserAgent = "Mozilla/5.0 (Windows NT 6.3; WOW64; rv:31.0) Gecko/20100101 Firefox/31.0"
             };
-            var document1 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-1.asp");
-            var document2 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-1b.asp");
-            var document3 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-2.asp");
-            var document4 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-3.asp");
-            var document5 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-4.asp");
-            var document6 = webGet.Load("https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-6.asp");
+            
+            // Generate all the needed URL's
+            string[] URLs = GenerateDataURLs(
+                "https://www.drafttek.com/2021-NFL-Mock-Draft/2021-NFL-Mock-Draft-Round-<PageNo>.asp",
+                "<PageNo>", 1, 6, new string[]{"b"}, new string[]{});
+
+            // Initialize documents and lists            
+            HtmlDocument[] Docs = new HtmlDocument[URLs.Length]; 
+            List<MockDraftPick>[] MockDreaftPickLists = new List<MockDraftPick>[URLs.Length];
```

Comment:
I noticed that MockDraftPicks is misspelled as MockDreaftPickLists - that should probably be updated, too.  Sorry for not noticing this previously.